docs/02_cleos/03_command-reference/system/system-regproducer.md

```diff
@@ -6,7 +6,7 @@ excerpt: "Register a new producer"
 - `account` _TEXT_ - The account to register as a producer
 - `producer_key` _TEXT_ - The producer's public key
 - `url` _TEXT_ - URL where info about producer can be found
-- `location` _UINT_ - Relative location for purpose of nearest neighbor scheduling
+- `location` _UINT_ - Relative location for purpose of nearest neighbor scheduling, e.g. ISO 3166-1 numeric
```

Comment:
This is **relative** location for the purpose of sorting. The ISO 3166-1 numeric values are not sorted by location.